pkg/security/rules/engine.go

```diff
@@ -183,6 +183,24 @@ func (e *RuleEngine) Start(ctx context.Context, reloadChan <-chan struct{}, wg *
 			}
 		}
 	}()
+
+	// Sending an heartbeat event every minute
+	wg.Add(1)
```

Comment:
could we move the heartbeat thing in the policy monitor ?

Reply:
Done in https://github.com/DataDog/datadog-agent/pull/19775/commits/46019af51cb7346badfc4354ae90e5c4e6a756a3